packages/top-app-bar-fixed/README.md

```diff
@@ -0,0 +1,138 @@
+# `<mwc-top-app-bar-fixed>` [![Published on npm](https://img.shields.io/npm/v/@material/mwc-top-app-bar-fixed.svg)](https://www.npmjs.com/package/@material/mwc-top-app-bar-fixed)
+
+> IMPORTANT: The Material Web Components are a work in progress and subject to
+> major changes until 1.0 release.
+
+Fixed Top App Bars are a container for items such as application title, navigation icon, and action items that are always visible.
+
+![](images/fixed.gif)
```

Comment:
The gifs are awesome, exactly what's needed to show the difference between the three layouts. I can't help myself notice these nits though...

- It looks very blurry on my retina screen (whereas the other screenshots below it look perfect). Was it recorded on a low dpi screen?
- I think it'd look better without the mouse cursor visible (put the mouse lower down so it's not in the crop?)
- I think it'd look better if the text was left padded, and probably a lower font weight (e.g. like in this screenshot from the spec?)

![image](https://user-images.githubusercontent.com/48894/63533729-3d244f00-c4c2-11e9-9c7f-2e0895288691.png)

Reply:
Didn't change the font weight, but a `16px` padding left and right looks a lot better.

Reply:
Looks better! Still looks blurry though.

Reply:
Hmm, this is only looking blurry on a high DPI display. I think the tool I'm using is just not capturing at 2x? Or maybe this is as good as I can get from it.

There are alternate options, but maybe this is acceptable for now and I can look into improving the gif quality in the future?

Reply:
Yeah looks like licecap might be bad at doing high dpi: https://github.com/justinfrankel/licecap/issues/9. Ok, if you don't want to spend more time on it, your call.